static/js/term.js

```diff
@@ -3510,7 +3510,7 @@ Terminal.prototype.deviceStatus = function(params) {
         break;
     }
   } else if (this.prefix === '?') {
-    // modern xterm doesnt seem to
+    // modern xterm doesn't seem to
```

Comment:
I'd drop the changes to this file as it's a verbatim copy from an external project and altering it will make it much harder for us to keep track of it.

Reply:
> I'd drop the changes to this file as it's a verbatim copy from an external project and altering it will make it much harder for us to keep track of it.

@stgraber Got it. Reverted.